pkg/ebpf/tracer.go

```diff
@@ -302,6 +302,8 @@ func inspectByPort(ctx context.Context, cfg *ebpfcommon.TracerConfig, functions
 		return nil, nil, fmt.Errorf("looking for executable ELF: %w", err)
 	}
 
+	pidMap := make(map[int32]exec.FileInfo)
```

Comment:
nitpick, idiomatic way of instantiating maps in Go is:
```go
pidMap := map[int32]exec.FileInfo{}
```

BTW, are the worker instances stable, or they are created and destroyed regularly? In that case, that would lead to a memory leak here. We should either:
* delete the pidMap entry when the process is destroyed
* probably easier, cleanup this map periodically
* probably even easier, use the [vendored LRU implementation](https://github.com/grafana/ebpf-autoinstrument/blob/main/vendor/github.com/hashicorp/golang-lru/v2/lru.go) with a maximum size that we know will be never surpassed by the number of active processes.